graylog2-web-interface/src/components/widgets/WidgetFooter.jsx

```diff
@@ -47,25 +47,16 @@ const WidgetFooter = React.createClass({
     }
 
     // if we have a tooltip, we disable the button link and instead show a tooltip on hover
-    const title = this.props.replayToolTip ? null : 'Replay search';
-    const href = this.props.replayToolTip ? null : this.props.replayHref;
-    let replay = (
-      <Button bsStyle="link" className="btn-text" title={title} href={href}>
-        <i className="fa fa-play" />
-      </Button>
+    const replay = this.props.replayDisabled ? null : (
+      <div className="widget-replay">
+        <Button bsStyle="link" className="btn-text" title={'Replay search'} href={this.props.replayHref}>
```

Comment:
In the `title` prop we should use double-quotes which read a bit better, e.g. `title="Replay search"`.